### ISSUE TITLE
🚀 Feature Request: global classes for widgets

### ISSUE THREAD

**Issue by 79ho3ein** [elementor/elementor#19531]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

hi elementor team!

I have already addressed this issue. But I didn't know much about it and it was just an initial idea. But now I found a similar item for it which can definitely be useful.
In fact, global classes are a wonderful feature that we can use to connect styles of similar widgets. For example, I have 10 similar looking button widgets on my website. Using this feature, we will link these 10 widgets together. If we change the style of only one of the 10 widgets, the other 9 widgets will be updated automatically. Everything changes incredibly fast.

I worked with bricks builder for the first time and saw that bricks have this feature(Of course, I love Elementor and will never leave it). Anyway, please watch the video below :

https://user-images.githubusercontent.com/77163248/185511560-2f2171ba-101c-48a9-a218-d97f7304cb48.mp4

**There are two important points regarding this wonderful feature.**

1- We can create multiple classes for each widget. (If needed)

2- if I create a class in the button widget, this class will only be displayed and usable in the button widget. In general, each widget displays its own classes.

**By creating a class, I mean the part that I saved as elementor in the video above.**

thank you

### Describe the solution you'd like

global classes for widgets. If you decide to add this feature, it's best to add it to the Widgets / Containers style tab > First section.

![download (2)](https://user-images.githubusercontent.com/77163248/185513258-b341aa69-049f-42c5-b4e5-1816d61cfbca.png)

**Comment by perks98** [comment]
I think that implementation would be a game changer if Elementor did that. So many global design possibilities for inheriting changes, especially as it looks like you can use more than one class to inherit values from. +1

**Comment by 79ho3ein** [comment]
> I think that implementation would be a game changer if Elementor did that. So many global design possibilities for inheriting changes, especially as it looks like you can use more than one class to inherit values from. +1

Yes, we can create multiple classes for each widget and there is no limit.

**Comment by Michaelgimii** [comment]
+1
Great feature. Elementor please...

**Comment by Michaelgimii** [comment]
Hi @arielk , May I ask if you are currently working on this feature? This is truly a game changer for Elementor.

**Comment by arielk** [comment]
@Michaelgimii We are looking at several solutions to solve the issue of global style editing...

**Comment by Michaelgimii** [comment]
thanks for the reply @arielk.2022 seems unlikely, but I hope it will be added to Elementor in 2023. thanks anyway

**Comment by nviitala** [comment]
+1, this would be a great feature

**Comment by Inventer89** [comment]
Is nov 2024 and still is not available
